ai2thor/tests/test_event.py

```diff
@@ -5,6 +5,7 @@
 import json
 import numpy as np
 import pytest
+from ai2thor.tests.constants import TESTS_DATA_DIR, TEST_SCENE
```

Comment:
Seems like `TESTS_DATA_DIR` is imported on line 10 again?

Reply:
good catch, removed.